elyra/pipeline/processor_kfp.py

```diff
@@ -511,7 +511,7 @@ def _cc_pipeline(self,
                             # If file can't be found locally, assume a remote file location was entered.
                             # This may cause the pipeline run to fail; the user must debug in this case.
                             pass
-                    elif component_property.type in ['dict', 'dictionary']:
+                    elif component_property.type in ['dictionary', 'list']:
```

Comment:
I believe there were situations where `dict` was being presented in earlier implementation. Are we sure it is not anymore? Or should we add `list` and still leave `dict`?

Reply:
You're right! That was the case. But in #1936, we sort of "standardized" the types that are assigned. So now if a component parameter has `dict` listed as its type in the spec, it's assigned the type `dictionary` instead on instantiation. Its in `component.py:354` `determine_type_information()`